apps/coscrad-frontend/src/components/resources/utils/render-media-length-in-minutes-cell.tsx

```diff
@@ -0,0 +1,5 @@
+export const renderMediaLengthInMinutes = (length): JSX.Element => {
+    const lengthInMinutes = Math.round(length / 1000);
+
+    return <div>{lengthInMinutes} Sec</div>;
```

Comment:
Why is `Sec` following length in **minutes**?

Reply:
LOL, didn't catch that.  I think I started with the idea of doing minutes but then realized that all the test clips are less than a minute...

Reply:
Fixed